.goreleaser.yml

```diff
@@ -3,10 +3,31 @@ archives:
   - files:
       - LICENSE
       - docs/cli_README.md
+    replacements:
+      amd64: 64bit
+      arm64: ARM64
+      darwin: macOS
+      linux: Linux
+      windows: Windows
+release:
+  draft: true
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^docs:'
+      - typo
+      - (?i)foo
 builds:
-  - env: [CGO_ENABLED=0]
+  - env: 
+    - CGO_ENABLED=0 
```

Comment:
Can you add the details about the cli binary size with and without this flag?

Reply:
@arajkumar Without the `CGO_ENABLED` flag, the binary size is 12.8 MB